sdk/python/express_relay/searcher/examples/simple_searcher_svm.py

```diff
@@ -240,6 +252,20 @@ async def main():
         default=100,
         help="How much of the initial order size to fill in percentage. Default is 100%",
     )
+    parser.add_argument(
```

Comment:
same here. ideally these should be in a separate script or at least we should document that these are just for internal testing and searchers do not need to implement them